compiler/rustc_driver_impl/src/lib.rs

```diff
@@ -1453,13 +1453,14 @@ mod signal_handler {
     /// When an error signal (such as SIGABRT or SIGSEGV) is delivered to the
     /// process, print a stack trace and then exit.
     pub(super) fn install() {
+        use std::alloc::{alloc, Layout};
+
         unsafe {
-            const ALT_STACK_SIZE: usize = libc::MINSIGSTKSZ + 64 * 1024;
+            let alt_stack_size: usize = min_sigstack_size() + 64 * 1024;
```

Comment:
It might seem like 64KiB is greater than the largest number I mentioned, 16KiB.

...but it's up to 16KiB *per vector register*, and it has way more than one. 🫠